views/layout.slim

```diff
@@ -1,5 +1,14 @@
 html
   head
     title Trainify
+    script src="https://code.jquery.com/jquery-1.12.4.js"
+    script src="https://code.jquery.com/ui/1.12.1/jquery-ui.js"
+    link rel="stylesheet" href="https://code.jquery.com/ui/1.12.1/themes/base/jquery-ui.css"
+    link rel="stylesheet" href="/stylesheets/styles.css"
+    link rel="stylesheet" href="/stylesheets/dashboard.css"
```

Comment:
You can dynamically load all stylesheets from that folder using a combination of `Dir.glob` + `each` + `File.basename`